ewatercycle/models/marrmot.py

```diff
@@ -259,3 +259,258 @@ def parameters(self) -> Iterable[Tuple[str, Any]]:
             ('forcing_file', self.forcing_file),
         ]
         return p
+
+
+M14_PARAMS = ['maximum_soil_moisture_storage',
+            'threshold_flow_generation_evap_change',
+            'leakage_saturated_zone_flow_coefficient',
+            'zero_deficit_base_flow_speed',
+            'baseflow_coefficient',
+            'gamma_distribution_chi_parameter',
+            'gamma_distribution_phi_parameter']
```

Comment:
Please use tuple for immutable variable. This makes sure that nobody can change it.
```suggestion
M14_PARAMS = ('maximum_soil_moisture_storage',
            'threshold_flow_generation_evap_change',
            'leakage_saturated_zone_flow_coefficient',
            'zero_deficit_base_flow_speed',
            'baseflow_coefficient',
            'gamma_distribution_chi_parameter',
            'gamma_distribution_phi_parameter')
```

Reply:
thanks, I didn't know about it.